### PR TITLE
Fix MessageCache and further Bugs

### PR DESCRIPTION
**Pull request by TureBentzin**
### This is a Bug-fixing pull request, that fixes a lot of crucial bugs and glitches with the Core

Among these fixes are the following:

- NullPointerException if caching is disabled and somebody tries to cache stuff
- Fallbacks are now working as expected, with replaces in foreign languages
- Server lags with too large locale (fixed)
- Message API can't send messages to multiple recipients with replaces (fixed)
- replaces are not replaced after a language switch to a language that has no translation for the message in the database (fixed)

### CODE REVIEW

File: API/src/main/java/net/juligames/core/api/jdbi/SQLManager.java [hunk 1]
```diff
@@ -40,6 +43,19 @@ public interface SQLManager {
...
 
+    /**
+     * You can use this to execute stuff on JDBI fast and secure
+     * @return a new Handle
+     * @see SQLManager#openHandle()
+     */
+    <R> R useHandle(Function<Handle,R> handleFunction);
+
+    @ApiStatus.Experimental
+    ResultIterable<Map<String, Object>> mapQuery(String sql);
```
Comment (Bommels05) [inline API/src/main/java/net/juligames/core/api/jdbi/SQLManager.java:54]: Interesting!

File: AdventureCore/src/main/java/net/juligames/core/adventure/BlankTag.removed [hunk 2]
```diff
@@ -0,0 +1,65 @@
+package net.juligames.core.adventure;
```
Comment (Bommels05) [inline AdventureCore/src/main/java/net/juligames/core/adventure/BlankTag.removed:1]:
```suggestion
.removed
```

**Review by Bommels05 — APPROVED** [review]
Hopefully the Core is now a bit more stable than before!

File: AdventureCore/src/main/java/net/juligames/core/adventure/BlankTag.removed (hunk 2 above)
Comment (TureBentzin) [reply AdventureCore/src/main/java/net/juligames/core/adventure/BlankTag.removed:1]: no

File: API/src/main/java/net/juligames/core/api/jdbi/SQLManager.java (hunk 1 above)
Comment (TureBentzin) [reply API/src/main/java/net/juligames/core/api/jdbi/SQLManager.java:54]: yes

**Pull request merged**